[CI] RcsCcsCommonYamlTestSuiteIT test {p0=search.vectors/200_dense_vector_docvalue_fields/dense_vector docvalues with bfloat16} failing

**Build Scans:**
- [elasticsearch-intake #32230 / part6](https://gradle-enterprise.elastic.co/s/fq3hmmpz6cddy)
- [elasticsearch-pull-request #108566 / part-6](https://gradle-enterprise.elastic.co/s/r5b6zaytifzt6)
- [elasticsearch-pull-request #108189 / part-6](https://gradle-enterprise.elastic.co/s/oyjr5q255vzko)
- [elasticsearch-pull-request #108178 / part-6](https://gradle-enterprise.elastic.co/s/gdlus2u3omstg)
- [elasticsearch-pull-request #108139 / part-6](https://gradle-enterprise.elastic.co/s/ixw4v3bepkdma)

**Reproduction Line:**
```
./gradlew ":qa:ccs-common-rest:yamlRestTest" --tests "org.elasticsearch.test.rest.yaml.RcsCcsCommonYamlTestSuiteIT" -Dtests.method="test {p0=search.vectors/200_dense_vector_docvalue_fields/dense_vector docvalues with bfloat16}" -Dtests.seed=C02122D63A7504D0 -Dtests.locale=ar-OM -Dtests.timezone=Europe/Luxembourg -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.test.rest.yaml.RcsCcsCommonYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bp0%3Dsearch.vectors%2F200_dense_vector_docvalue_fields%2Fdense_vector%20docvalues%20with%20bfloat16%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: org.junit.internal.runners.model.MultipleFailureException: There were ٢ errors:
  java.lang.AssertionError(
Expected: a numeric value within <0.4> of <100.0>
     but: <99.54383> differed by <0.056170000000000164> more than delta <0.4>)
  java.lang.AssertionError(
Expected: a numeric value within <0.4> of <-156.0>
     but: <-155.49182> differed by <0.10818000000001005> more than delta <0.4>)
```

**Issue Reasons:**
- [main] 5 failures in test test {p0=search.vectors/200_dense_vector_docvalue_fields/dense_vector docvalues with bfloat16} (0.6% fail rate in 883 executions)
- [main] 4 failures in step part-6 (0.8% fail rate in 494 executions)
- [main] 4 failures in pipeline elasticsearch-pull-request (0.8% fail rate in 527 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -481,9 +481,6 @@ tests:
 - class: org.elasticsearch.multiproject.test.CoreWithMultipleProjectsClientYamlTestSuiteIT
   method: test {yaml=search.retrievers/result-diversification/10_mmr_result_diversification_retriever/Test MMR result diversification single index float type}
   issue: https://github.com/elastic/elasticsearch/issues/139335
-- class: org.elasticsearch.test.rest.yaml.RcsCcsCommonYamlTestSuiteIT
-  method: test {p0=search.vectors/200_dense_vector_docvalue_fields/dense_vector docvalues with bfloat16}
-  issue: https://github.com/elastic/elasticsearch/issues/139368
 
 # Examples:
 #

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/200_dense_vector_docvalue_fields.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/200_dense_vector_docvalue_fields.yml
@@ -298,17 +298,17 @@ setup:
   - length: {hits.hits.0.fields.vector7.0: 5}
   - length: {hits.hits.0.fields.vector8.0: 5}
 
-  - close_to: { hits.hits.0.fields.vector7.0.0: { value: 230.0, error: 0.4 } }
-  - close_to: { hits.hits.0.fields.vector7.0.1: { value: 300.33, error: 0.4 } }
-  - close_to: { hits.hits.0.fields.vector7.0.2: { value: -34.8988, error: 0.4 } }
-  - close_to: { hits.hits.0.fields.vector7.0.3: { value: 15.555, error: 0.4 } }
-  - close_to: { hits.hits.0.fields.vector7.0.4: { value: -200.0, error: 0.4 } }
+  - close_to: { hits.hits.0.fields.vector7.0.0: { value: 230.0, error: 1 } }
+  - close_to: { hits.hits.0.fields.vector7.0.1: { value: 300.33, error: 1 } }
+  - close_to: { hits.hits.0.fields.vector7.0.2: { value: -34.8988, error: 0.2 } }
+  - close_to: { hits.hits.0.fields.vector7.0.3: { value: 15.555, error: 0.2 } }
+  - close_to: { hits.hits.0.fields.vector7.0.4: { value: -200.0, error: 1 } }
 
-  - close_to: { hits.hits.0.fields.vector8.0.0: { value: 130.0, error: 0.4 } }
-  - close_to: { hits.hits.0.fields.vector8.0.1: { value: 115.0, error: 0.4 } }
-  - close_to: { hits.hits.0.fields.vector8.0.2: { value: -1.02, error: 0.4 } }
-  - close_to: { hits.hits.0.fields.vector8.0.3: { value: 15.555, error: 0.4 } }
-  - close_to: { hits.hits.0.fields.vector8.0.4: { value: -100.0, error: 0.4 } }
+  - close_to: { hits.hits.0.fields.vector8.0.0: { value: 130.0, error: 1 } }
+  - close_to: { hits.hits.0.fields.vector8.0.1: { value: 115.0, error: 1 } }
+  - close_to: { hits.hits.0.fields.vector8.0.2: { value: -1.02, error: 0.2 } }
+  - close_to: { hits.hits.0.fields.vector8.0.3: { value: 15.555, error: 0.2 } }
+  - close_to: { hits.hits.0.fields.vector8.0.4: { value: -100.0, error: 1 } }
 
   - match: {hits.hits.1._id: "2"}
   - match: {hits.hits.1.fields.name.0: "moose.jpg"}
@@ -316,11 +316,11 @@ setup:
   - length: {hits.hits.1.fields.vector7.0: 5}
   - match: {hits.hits.1.fields.vector8: null}
 
-  - close_to: { hits.hits.1.fields.vector7.0.0: { value: -0.5, error: 0.4 } }
-  - close_to: { hits.hits.1.fields.vector7.0.1: { value: 100.0, error: 0.4 } }
-  - close_to: { hits.hits.1.fields.vector7.0.2: { value: -13, error: 0.4 } }
-  - close_to: { hits.hits.1.fields.vector7.0.3: { value: 14.8, error: 0.4 } }
-  - close_to: { hits.hits.1.fields.vector7.0.4: { value: -156.0, error: 0.4 } }
+  - close_to: { hits.hits.1.fields.vector7.0.0: { value: -0.5, error: 0.2 } }
+  - close_to: { hits.hits.1.fields.vector7.0.1: { value: 100.0, error: 1 } }
+  - close_to: { hits.hits.1.fields.vector7.0.2: { value: -13, error: 0.2 } }
+  - close_to: { hits.hits.1.fields.vector7.0.3: { value: 14.8, error: 0.2 } }
+  - close_to: { hits.hits.1.fields.vector7.0.4: { value: -156.0, error: 1 } }
 
   - match: {hits.hits.2._id: "3"}
   - match: {hits.hits.2.fields.name.0: "rabbit.jpg"}
@@ -328,8 +328,8 @@ setup:
   - length: {hits.hits.2.fields.vector8.0: 5}
   - match: {hits.hits.2.fields.vector7: null}
 
-  - close_to: { hits.hits.2.fields.vector8.0.0: { value: 130.0, error: 0.4 } }
-  - close_to: { hits.hits.2.fields.vector8.0.1: { value: 115.0, error: 0.4 } }
-  - close_to: { hits.hits.2.fields.vector8.0.2: { value: -1.02, error: 0.4 } }
-  - close_to: { hits.hits.2.fields.vector8.0.3: { value: 15.555, error: 0.4 } }
-  - close_to: { hits.hits.2.fields.vector8.0.4: { value: -100.0, error: 0.4 } }
+  - close_to: { hits.hits.2.fields.vector8.0.0: { value: 130.0, error: 1 } }
+  - close_to: { hits.hits.2.fields.vector8.0.1: { value: 115.0, error: 1 } }
+  - close_to: { hits.hits.2.fields.vector8.0.2: { value: -1.02, error: 0.2 } }
+  - close_to: { hits.hits.2.fields.vector8.0.3: { value: 15.555, error: 0.2 } }
+  - close_to: { hits.hits.2.fields.vector8.0.4: { value: -100.0, error: 1 } }

